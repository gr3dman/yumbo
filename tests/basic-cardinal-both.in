[foo]1-0..*[bar]
