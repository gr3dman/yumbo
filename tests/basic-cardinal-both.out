table foo
one
zero or many
table bar

